org.eclipse.lemminx/pom.xml

```diff
@@ -246,6 +246,17 @@
 			<version>1.2</version>
 			<scope>runtime</scope>
 		</dependency>
+        <dependency>
+            <groupId>org.relaxng</groupId>
+            <artifactId>jing</artifactId>
+            <version>20220510</version>
```

Comment:
After talking to @datho7561 , I noticed that https://repo1.maven.org/maven2/org/relaxng/jing/20220510/jing-20220510.pom depends on :

`xerces:xercesImpl:2.9.1` & `xml-apis:xml-apis:1.0.b2`

while lemminx itself depends on :

`xerces:xercesImpl:2.12.2` & `xml-apis:xml-apis:1.4.01`

We can't have multiple versions in the uber jar so would be good to ensure the latest versions are also supported by jing, and maybe contribute back upstream.